src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -908,18 +909,27 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
   if (this->title.empty())
     this->title = "3D Scene";
 
+  std::string cmdRenderEngineGUI = gui::renderEngineName();
   // Custom parameters
   if (_pluginElem)
   {
     auto elem = _pluginElem->FirstChildElement("engine");
-    if (nullptr != elem && nullptr != elem->GetText())
+    if (cmdRenderEngineGUI.empty())
+    {
+      if (nullptr != elem && nullptr != elem->GetText())
+      {
+        renderWindow->SetEngineName(elem->GetText());
+        // there is a problem with displaying ogre2 render textures that are in
+        // sRGB format. Workaround for now is to apply gamma correction
+        // manually.
+        // There maybe a better way to solve the problem by making OpenGL calls.
+        if (elem->GetText() == std::string("ogre2"))
+          this->PluginItem()->setProperty("gammaCorrect", true);
```

Comment:
Should this be moved out of this `if`? We need to set `gammaCorrect` for `ogre2` regardless of whether the engine name came from the window property or the XML.